examples/event-migration/build.sbt

```diff
@@ -0,0 +1,26 @@
+name := "event-migration"
+version := "0.1"
+scalaVersion := "2.13.6"
```

Comment:
Pls bump from 2.13.5 in the other places as well

Reply:
#70